lang/ir/src/ir/item/storage.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright 2018-2020 Parity Technologies (UK) Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::ir;
+use core::convert::TryFrom;
+use proc_macro2::Ident;
+use syn::spanned::Spanned as _;
+
+/// An ink! storage struct definition.
+///
+/// Noticed by ink! throught the `#[ink(storage)]` annotation.
+///
+/// # Note
+///
+/// An ink! smart contract must have exactly one storage definition.
+/// The storage definition must be found in the root of the ink! module.
+///
+/// # Example
+///
+/// ```
+/// # use core::convert::TryFrom;
+/// # let event = <ink_lang_ir::Storage as TryFrom<syn::ItemStruct>>::try_from(syn::parse_quote! {
```

Comment:
```suggestion
/// # let storage = <ink_lang_ir::Storage as TryFrom<syn::ItemStruct>>::try_from(syn::parse_quote! {
```

Reply:
let us simply remove the let binding altogether.